patches/react-native-keyboard-aware-scroll-view+0.8.8-wp-1.patch

```diff
@@ -0,0 +1,76 @@
+diff --git a/node_modules/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js b/node_modules/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js
+index 30f62c9..83a6920 100644
+--- a/node_modules/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js
++++ b/node_modules/react-native-keyboard-aware-scroll-view/lib/KeyboardAwareHOC.js
+@@ -264,9 +264,13 @@ function KeyboardAwareHOC(
+       })
+     }
+ 
+-    componentWillReceiveProps(nextProps: KeyboardAwareHOCProps) {
+-      if (nextProps.viewIsInsideTabBar !== this.props.viewIsInsideTabBar) {
+-        const keyboardSpace: number = nextProps.viewIsInsideTabBar
++    // This patch changed from the deprecated `componentWillReceiveProps` to
++    // `componentDidUpdate`. We can remove this patch when we upgrade to
++    // `react-native-keyboard-aware-scroll-view@^0.9.2`
```

Comment:
Hmm, the componentWillReceiveProps -> componentDidUpdate migration happened two years ago in upstream. Feels we should prioritize upgrading from upstream soon. Would mind opening a ticket about this @dcalhoun ? Actually, wdyt about an umbrella ticket with "follow up items" since we start to have some of those already? No need for individual tickets for each thing yet.

Reply:
(and there are more forks below that seem fairly outdated so, perhaps an item to remind us to make a pass over all and upgrade them will be useful)

Reply:
Agreed, IMO we should manage third-party dependencies in the same manner we manage React Native. I.e. we audit and upgrade them on some cadence. Otherwise, it is very likely they will fall behind and become increasing difficult to upgrade. 

I created https://github.com/WordPress/gutenberg/issues/37594 as an "umbrella" ticket for all tasks that arose from this PR.